examples/src/simple.ts

```diff
@@ -26,10 +37,58 @@ async function main() {
   const renderer = new THREE.WebGLRenderer();
   renderer.setClearColor('#444');
   renderer.setSize(window.innerWidth, window.innerHeight);
+
+  // Debug overlay for "determineSectors"
+  const canvas = coverageUtil.createDebugCanvas({ width: 160, height: 100 });
+  canvas.style.position = 'fixed';
+  canvas.style.left = '8px';
+  canvas.style.top = '8px';
+  document.body.appendChild(canvas);
   document.body.appendChild(renderer.domElement);
 
   const { position, target, near, far } = cadNode.suggestCameraConfig();
-  const camera = new THREE.PerspectiveCamera(75, window.innerWidth / window.innerHeight, near, far);
+  camera.near = near;
+  camera.far = 3 * far;
+  camera.fov = 75;
+  camera.aspect = window.innerWidth / window.innerHeight;
+  camera.updateProjectionMatrix();
+  camera.updateMatrixWorld();
+
+  document.addEventListener('keypress', event => {
```

Comment:
Move to helper class

Reply:
Created a separate example for this (short lived - will be the one and only culler when we get the new fix from J)